src/components/Widgets/Widget.style.ts

```diff
@@ -13,15 +13,98 @@ export const WidgetWrapper = styled(Box, {
   width: '100%',
   position: 'relative',
   margin: theme.spacing(0, 'auto'),
-  ...(!welcomeScreenClosed && {
-    '&:hover': {
-      marginTop: 0,
-    },
-  }),
   zIndex: 2,
-  // [theme.breakpoints.up('sm' as Breakpoint)]: {
-  //   maxWidth: 416,
-  // },
+
+  '> div': {
```

Comment:
Not fan of this as it is only related to welcomeScreen, can you add it to the welcomescreen component itself?

Reply:
I don´t think we can move this easily to the WelcomeScreen itself as they´re not directly related in the tree as children/parents.

This ugly hack allows to directly style the widget via the widget-wrapper. It´s mainly needed for positioning.
If you know of a cleaner way, I´d love to hear. Maybe the "container" prop within the widgetTheme is a better choice??